tmp/build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -0,0 +1,212 @@
+    mediatype: image/png
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+      - serviceAccountName: istio-operator
+        rules:
+__CLUSTER_ROLE_RULES__
+      deployments:
+      - name: istio-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: istio-operator
+          template:
+            metadata:
+              labels:
+                name: istio-operator
+            spec:
+              serviceAccountName: istio-operator
+              volumes:
+              - name: discovery-cache
+                emptyDir:
+                  medium: Memory
+              containers:
+                - name: istio-operator
+                  image: __IMAGE_SRC__
+                  ports:
+                  - containerPort: 60000
+                    name: metrics
+                  command:
+                  - istio-operator
+                  - --discoveryCacheDir
+                  - /home/istio-operator/.kube/cache/discovery
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      value: ""
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: OPERATOR_NAME
+                      value: "istio-operator"
+                  volumeMounts:
+                  - name: discovery-cache
+                    mountPath: /home/istio-operator/.kube/cache/discovery
+  customresourcedefinitions:
+    required:
+    - name: kialis.kiali.io
+      version: v1alpha1
+      kind: Kiali
+      displayName: Kiali
+      description:  A configuration file for a Kiali installation.
+    - name: jaegers.jaegertracing.io
+      version: v1
+      kind: Jaeger
+      displayName: Jaeger
+      description: A configuration file for a Jaeger installation.  
+    owned:
+    - name: controlplanes.istio.openshift.com
```

Comment:
Remove this as well?

Reply:
I'd like to keep this for now, as I won't be able to test generated manifests without this until Rob's changes are in.